INSTALLATION.md

```diff
@@ -57,7 +57,22 @@ TODO
 
 ### macOS
 
-TODO
+A current version of VAST can be installed using Homebrew.
+
+Install VAST from tenzir/tenzir tap (current master):
```

Comment:
It's a matter of taste, but I'd make it more clear what gets installed and put the install target first:
```suggestion
Install the *current master* from tenzir/tenzir tap:
```